plugins/pga-tour-leaderboard/manager.py

```
@@ -0,0 +1,1345 @@
+                return
+
+            today = datetime.now()
+
+            # Search backwards day by day to find most recent completed tournament
+            # Check every 3 days to balance thoroughness with API calls
+            for days_back in range(1, 31, 3):  # Check 1, 4, 7, 10, 13...28 days back
+                date_to_check = today - timedelta(days=days_back)
+                date_str = date_to_check.strftime('%Y%m%d')
+
+                data = self.api_helper.get(
+                    url=f"https://site.api.espn.com/apis/site/v2/sports/golf/pga/scoreboard",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)